src/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = class BlockService {
   }
 
   isOnline () {
-    return this._bitswap != null
+    return this._bitswap !== null
```

Comment:
Well.. you are always setting it up to null, if it happens to be undefined then probably something weird happened, but ok, I can revert